### DISCUSSION TITLE
Irish music for St. Patty's Day

### DISCUSSION THREAD

**Discussion by BenjaminGolden** [nss-day-cohort-47/C47-Discussions#61] [Topix.random]
Hi guys! I recorded a new video for St. Patrick's Day and I wanted to share it with y'all. I hope it adds some Irish flavor to your day. Happy coding!

https://youtu.be/r4f8t0LrhNM

**Comment by ardisrtilson** [comment]
This song is actually sorta sacred in Boston

https://www.youtube.com/watch?v=x-64CaD8GXw

**Reply by BenjaminGolden** [reply]
absolutely!

**Comment by D-Upchurch** [comment]
Man that was fantastic, thanks for sharing!

**Reply by BenjaminGolden** [reply]
Thank you, Dakota! Thanks for taking the time to check it out! I appreciate it.
